tests/test_main.py

```diff
@@ -5,25 +5,19 @@
 from .fixtures import fixtures_globals as constants
 
 
-@pytest.fixture(autouse=True)
```

Comment:
Let's leave this autouse fixture here to avoid unused auth_mock every where else